tasks/README.md

```diff
@@ -1,99 +1,100 @@
+`task086_translated_symbol_arithmetic`	|  Performing Arithmetic with translated operator symbols.	| Arithmetic
+`task087_new_operator_addsub_arithmetic`	|  Performing Arithmetic with newly defined operator symbols.	| Arithmetic
+`task088_identify_typo_verification`	|  Identifying typo in a sentence.	| Verification
+`task089_swap_words_verification`	|  Identifying swapped words in a sentence.	| Verification
+`task090_equation_learner_algebra`	|  Answering based on the given equation.	| Algebra
+`task092_check_prime_classification`	|  Finding whether the number is prime or not.	| Mathematics
+`task079_conala_concat_strings` | Given a list of strings concatenate them to form one string | Answer Generation
+`task082_babi_t1_single_supporting_fact_question_generation` | Generating a question, given a collection of facts | Question Generation  
+`task083_babi_t1_single_supporting_fact_answer_generation` | Generating an answer, given a collection of evidence sentences | Answer Generatiomn 
+`task084_babi_t1_single_supporting_fact_identify_relevant_fact` | Given a question and answer, identifying the relevant piece of evidence | Supporting Fact Identification  
+`task093_conala_normalize_lists` | Given a list of numbers normalize the list such that the result adds to 1 | Answer Generation, Arithmetic
+`task094_conala_calculate_mean` | Given a list of numbers calculate the mean of the list | Answer Generation, Arithmetic
+`task095_conala_max_absolute_value` | Given a list of numbers calculate the element with the largest absolute value | Answer Generation, Arithmetic
+`task096_conala_list_index_subtraction` | Given a list of numbers subtract each element by its index in the list | Answer Generation, Arithmetic
+`task097_conala_remove_duplicates` | Given a list of numbers remove all of the duplicates in the list | Text Modification, Arithmetic
+`task098_conala_list_intersection` | Given a two lists of numbers find the intersection of the two lists | Answer Generation, Arithmetic
+`task_111_asset_sentence_simplification` | Given a sentence, simplify it so it can be understood by non-native English speakers | Generation, Paraphrasing
+`task_112_asset_simple_sentence_identification` | Given two text pieces, choose the one that is simpler and easier to understand by non-native speakers | Answer Generation, Sentence Comparison
+`task102_commongen_sentence_generation` | Given a collection of concepts, use them in a coherent sentence. | Sentence Generation 
+`task103_facts2story_long_text_generation` | Given 5 facts, write a story that incorporates them. | Long Text Generation 
+`task107_splash_question_to_sql` | Generate an SQL statement from a question asking for certain data. | Structured Query Generation
+`task108_ContextualAbuseDetection_classification` | Given a text detect whether it's abusive or not. | Classification
+`task109_SMSspamcollection_SpamSMSdetection` | Classify SMS into spam or ham | Classification
```

Comment:
I see that you have added this last line. I am troubled that it shows the whole file as changed.